util/mock/context.go

```diff
@@ -224,7 +224,8 @@ func NewContext() *Context {
 		ctx:         ctx,
 		cancel:      cancel,
 	}
-	sctx.sessionVars.MaxChunkSize = 2
+	sctx.sessionVars.InitChunkSize = 2
+	sctx.sessionVars.MaxChunkSize = 32
```

Comment:
Please keep the `MaxChunkSize` to 2. Many of the unit tests use this value to test the multiply returned Chunk behavior.